graphql/typeDefs.ts

```diff
@@ -170,8 +170,12 @@ export default gql`
   type Star {
     id: String!
     studentId: Int
+    studentUsername: String
+    studentName: String
     mentorId: Int
     lessonId: Int
+    lessonTitle: String
+    lessonDifficulty: Int
```

Comment:
`lessonDifficulty` implies that some lessons are easier than other. Is this just `lessonId`? (eg. `Foundations` lesson has id `5`, Variables lesson has id `2` and etc?)

Reply:
ah, it's actually the 'order' of the lesson. the lesson IDs are kinda all over the place.